ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/transaction/UtxoTransactionBuilder.kt

```diff
@@ -5,6 +5,7 @@ import net.corda.v5.base.annotations.DoNotImplement
 import net.corda.v5.base.annotations.Suspendable
 import net.corda.v5.crypto.SecureHash
 import net.corda.v5.ledger.common.Party
+import net.corda.v5.ledger.utxo.Attachment
```

Comment:
Why do we need the attachment import here?

Reply:
@MrMatthewLayton  ?

Reply:
It seems to fix intellij's linking in the `addAttachment()`'s kdoc.

Reply:
Should I remove it?

Reply:
@blsemo I'm to blame here, and I agree it's a bit insane. I tend to resolve yellow squiggly underlines where possible. Perhaps (and this is a wild stab in the dark), the import is needed so that when HTML based documentation is generated, it knows where to point the href for the link...

For example, if you look at the docs for `StateRef` below, the `index` component is an `Int`, and if you click it, it links to the Kotlin docs.

https://docs.r3.com/en/api-ref/corda/4.8/open-source/kotlin/corda/net.corda.core.contracts/-state-ref/index.html